concordium-consensus/src/Concordium/GlobalState/Persistent/Account/StructureV1.hs

```diff
@@ -1019,6 +1022,8 @@ data PersistentAccount av = PersistentAccount
       -- | The actively staked balance of the account.
       --  INVARIANT: This is 0 if the account is not a baker or delegator.
       accountStakedAmount :: !Amount,
+      -- | The state table of the protocol level tokens of the account in ascending order of the TokenIndex.
+      accountTokenStateTable :: !(Conditionally (SupportsPLT av) (HashedBufferedRef TokenAccountStateTable)),
```

Comment:
I think it might be better to use `Nullable (HashedBufferedRef TokenAccountStateTable)` with `Null` being used for the empty account state table, since this will reduce storage overhead for the many accounts that have no PLTs. I'd be fine with opening an issue for this instead of addressing it in this PR.

Reply:
It may also be useful to use an `EagerlyHashedBufferedRef` so that it is never necessary to read from the blob store to compute the hash of an account that is loaded (i.e. there can be a straight `HashableTo` implementation). This can also be deferred if we create an issue to reassess exactly how we want to hash and store the PLT state on an account.

Reply:
I created #1373 and #1374 for this.